monitorcontrol/__main__.py

```diff
@@ -49,6 +49,29 @@ def get_parser() -> argparse.ArgumentParser:
     group.add_argument(
         "--version", action="store_true", help="Show the version and exit."
     )
+    group.add_argument(
+        "--get-input-source",
+        action="store_true",
+        help="Get the input source of each monitor.",
+    )
+    group.add_argument(
+        "--set-input-source",
+        choices=[mode.name for mode in InputSource],
+        help="Set the input source of all monitors.",
+    )
+    group.add_argument(
+        "--get-monitors",
+        action="store_true",
+        help="Get the monitors.",
+    )
+    group = parser.add_argument_group("Optional monitor select")
+    group.add_argument(
+        "--monitor",
+        type=int,
+        default=0,
+        help="Select monitor for command. "
+        "Default: getters use monitor 1. Setters use all monitors.",
```

Comment:
Could this be done for setters as well?

Maybe a `default=None`, and then `None` does all monitors?

Reply:
Do you mean make the setters default to monitor 1?
Ill change it to default = None